suggestions/radios.json

```diff
@@ -1 +1,16 @@
-[]
\ No newline at end of file
+[
+    {
+        "id": "org.y20k.transistor",
+        "label": "Transistor",
+        "source": "f",
+        "reason": "Transistor is a really minimalistic app for listening to radio stations over the internet",
+        "repo": "https://codeberg.org/y20k/transistor"
+    },
+    {
+        "id": "community.peers.internetradio",
+        "label": "Internet Radio",
+        "source": "f",
+        "reason": "Small application to search Internet radio stations, and listen to them.\nRadio stations list maintained by RadioBrowser (https://www.radio-browser.info/)",
+        "repo": "https://notabug.org/zplus/InternetRadio"
+    }
```

Comment:
Require reviews. I think RadioDroid offers better features (e.g. recording and filtering) than both but it's not maintained anymore. I use a customised fork for my own use.

Reply:
**RadioDroid** is back again. Although the UI is not as modern as the others, the features it offers are more appealing that these two.

Reply:
- Transistor : https://codeberg.org/y20k/transistor#how-can-i-add-a-radio-station Adding new station is not easy for new users.
- Internet Radio : Dead project (last update 2019)

https://www.radio-browser.info/users also marked RadioDroid **recommended** in their list.